src/transforms/parallel.jl

```diff
@@ -18,6 +18,21 @@ struct Parallel <: Transform
   transforms::Vector{Transform}
 end
 
+function Base.show(io::IO, transform::Parallel)
+  print(io, transform.transforms[1])
+  for t in transform.transforms[2:end]
+    print(io, " ⊔ $t")
+  end
```

Comment:
I think we can use the 3-argument version of join to create the final string and then use a single println.

Reply:
Also, what about showing parallel transforms as a stack of transforms? I wonder if we could leverage some of the pretty printing packages to create more interesting visuals for these pipelines.